packages/babel-helper-replace-supers/src/index.js

```diff
@@ -227,7 +227,7 @@ export default class ReplaceSupers {
           t.variableDeclarator(ref, node.left),
         ]),
         t.expressionStatement(t.assignmentExpression("=", node.left,
-          t.binaryExpression(node.operator[0], ref, node.right))),
+          t.binaryExpression(node.operator.substring(0, node.operator.length - 1), ref, node.right))),
```

Comment:
With the exponentiation assignment operator (`**=`), AssignmentExpression operators can now be three characters. So instead of just taking the first character, we need to strip the equals sign off the end. 

This is what was making the `object-super` transform treat the exponentiation assignment operator like a multiplication assignment operator.